src/FamilyHubs.Referral.Core/Commands/CreateReferral/CreateReferralCommand.cs

```diff
@@ -118,26 +108,29 @@ private Data.Entities.Referral AttachExistingStatus(Data.Entities.Referral entit
         {
             entity.Status = referralStatus;
         }
+
         return entity;
     }
 
     private Data.Entities.Referral AttachExistingUserAccount(Data.Entities.Referral entity)
     {
         UserAccount? professional = _context.UserAccounts.SingleOrDefault(x => x.Id == entity.UserAccount.Id);
-        if (professional != null) 
+        if (professional != null)
         {
             entity.UserAccount = professional;
         }
         else
         {
-            if (entity.UserAccount != null && entity.UserAccount.UserAccountRoles != null) 
+            if (entity.UserAccount != null && entity.UserAccount.UserAccountRoles != null)
             {
                 for (int i = 0; i < entity.UserAccount.UserAccountRoles.Count; i++)
                 {
-                    Role? role = _context.Roles.SingleOrDefault(x => x.Name == entity.UserAccount.UserAccountRoles[i].Role.Name);
+                    Role? role =
+                        _context.Roles.SingleOrDefault(x => x.Name == entity.UserAccount.UserAccountRoles[i].Role.Name);
```

Comment:
nitpick and I'm not going to comment on all of them but the formatting may have gone a little overkill

Reply:
I've just set the line length to be longer, my screen generally has space for may more than 120 chars but maybe not everyone's does 🤷‍♂️ 

Approved anyway :)